ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -340,8 +350,7 @@ def handleAppletStateUpdateRequested(self):
         opDataExport = self.dataExportApplet.topLevelOperator
 
         # If no data, nothing else is ready.
-        input_ready = len(opDataSelection.ImageGroup) > 0 and not self.dataSelectionApplet.busy
-
+        input_ready = self._inputReady(2)
```

Comment:
Please add a check here, whether the `DataSelectionApplet` is busy, in order to prevent enabling the applets during stack imports